assembl/static2/css/views/idea.scss

```diff
@@ -205,7 +226,7 @@
   }
 }
 
-@media screen and (max-width: 600px) {
+@media screen and (max-width: 925px) {
```

Comment:
Ahhh finally caught this 925px ! please use a variable instead and add it to the list of existing variables.  Better, you should use one existing break point variables close to this value (if existing).

Reply:
Here we need to use a special breakpoint equal to the width of the post or we have responsive problems. I think the breakpoint will be used only here for the thread.